src/api-service/__app__/onefuzzlib/notifications/main.py

```diff
@@ -108,9 +108,12 @@ def get_queue_tasks() -> Sequence[Tuple[Task, Sequence[str]]]:
 
 
 def new_files(container: Container, filename: str) -> None:
-    report = get_report_or_regression(container, filename)
-
     notifications = get_notifications(container)
+
+    report = get_report_or_regression(
```

Comment:
since we're going to try to inspect every new file as a potential report, this sets up the parsing (seen later in this PR) to only log errors if we actually expect the files to be a report, rather than just optimistically being a report.